packages/management/addons-manager/README.md

```diff
@@ -0,0 +1,25 @@
+# addons-manager Package
```

Comment:
can you update the README with addons-manage information?

Reply:
added README file for addons-manager